src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  &__imac-photo {
+    display: block;
+    width: 100%;
```

Comment:
try to add width as in mockup< in this case you can delete padding: 0 3px;, I think

Reply:
Anya helped me to pass some tests with these paddings, so let it be)